src/browser/cached_resource_fetcher.ts

```diff
@@ -10,6 +10,7 @@ import { AuthCallback, Identity } from '../shapes';
 import { getSession } from './core_state';
 
 let appQuiting: Boolean = false;
+let clearCache: boolean = false;
```

Comment:
:neckbeard: Do both `Boolean` and `boolean` work in Typescript?

Reply:
both work, but the upper-cased `Boolean` should never be used:
https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html